src/Uplink/Admin/Provider.php

```diff
@@ -150,11 +150,11 @@ public function store_admin_notices( $page ): void {
 	 *
 	 * @since 1.0.0
 	 *
-	 * @param bool         $reply      Whether to bail without returning the package.
-	 *                                 Default false.
-	 * @param string       $package    The package file name or URL.
-	 * @param \WP_Upgrader $upgrader   The WP_Upgrader instance.
-	 * @param array        $hook_extra Extra arguments passed to hooked filters.
+	 * @param  bool|\WP_Error  $reply       Whether to bail without returning the package.
+	 *                                      Default false.
+	 * @param  string          $package     The package file name or URL.
+	 * @param  \WP_Upgrader    $upgrader    The WP_Upgrader instance.
+	 * @param  array           $hook_extra  Extra arguments passed to hooked filters.
```

Comment:
Two spaces between `@param` and the type?

Reply:
It's equal alignment docblock alignment, like the method below: https://github.com/stellarwp/uplink/blob/ec4b0970ed95202c923dc9df2e1abc2b95b9c252/src/Uplink/Admin/Provider.php#L179-L191